src/OrbitQt/main.cpp

```diff
@@ -84,6 +85,11 @@ void RunUiInstance(const DeploymentConfiguration& deployment_configuration,
 
   std::optional<orbit_qt::TargetConfiguration> target_config;
 
+  ErrorMessageOr<orbit_metrics_uploader::MetricsUploader> metrics_uploader = ErrorMessage("");
```

Comment:
Instead of `ErrorMessageOr`, could you use `std::optional<MetricsUploader>` for holding the instance? When it's `ErrorMessageOr` I ask myself where the error message is used. But it isn't which is a bit suprising.